### ISSUE TITLE
Use vibrant search window background, larger shadow & more rounded corners

### ISSUE THREAD

**Issue by sebj** [laconalabs/LaconaApp#140]
Depends where you're going with Lacona, but the only thing that bothers me (as functionality is pretty perfect right now) is appearance.

I'd love it if Lacona used a vibrant window background that shows background colour through as Spotlight does. Similarly, matching the larger shadow radius used by Spotlight, and all macOS apps would be great.

More rounded corners (like Spotlight) are a matter of preference, but aesthetically I personally prefer them.

As a side note, I realise I mentioned customisation options in #108, but.. a font size slider, the ability to turn off underlines (which I find immensely messy and annoying!) and fade/hide text for a more minimal look (again, like Spotlight) would be amazing.

Latest update is brilliant though, wasn't expecting so many updates – thanks!

**Comment by brandonhorst** [comment]
Transparency is discussed in https://github.com/laconalabs/LaconaApp/issues/53. Unfortunately, with the way that Lacona is currently implemented, it's not possible - it would require a rewrite of the rendering engine. I agree that it would make things look way better, but it's unfortunately not going to be coming soon.

As far as the shadows go, the smaller shadow is actually the Apple-recommended style for "Panel"-style windows. The XCode started window has these shadows, for example. I'm not entirely sure why Apple chose to use the full-style shadow for Spotlight, but I probably won't be changing this. 
![image](https://cloud.githubusercontent.com/assets/1938545/20576488/242cb7b8-b18c-11e6-8121-3aac11f49637.png)

The rounded corners will stay unless I have a good reason to change them.

I don't see myself adding more customization between the 3 "theme" options that are already presented. It could be possible to remove underlines in the "Simple Colors" theme, but I find them very helpful for differentiating between input and placeholders. I'd welcome other feedback on this.

As for "fading/hiding text", what did you have in mind there?

**Comment by sebj** [comment]
Fair enough on your reasoning, and apologies for duplicating an existing issue on transparency (and moving the window).

Spotlight may use higher shadow & corner radii as it's floating a level above windows and other on-screen items, or at least that'd be my reasoning. Considering how large the search box is with such little text, I think it may look better though.

On fading/hiding text, the small labels above tokens indicating type (application, preference pane etc) feel too prominent (I'd really like to hide them altogether). Most of the time I'm opening apps/preference panes/files/etc (e.g. just "saf" then enter for Safari etc) so hiding the "open" if I leave out an action verb would reduce clutter, or Lacona could default to hiding top automatic prefix.

Otherwise, removing the underline and fading underlined text with a background (image attached) might look less cluttered. 
![Underline suggestion](https://cloud.githubusercontent.com/assets/379991/20584008/87edac80-b1e6-11e6-80a4-35c111bf455b.png)

Generally, I want something as minimal as possible whilst giving me a good amount of information (currently enabled 'Simple Colors'; no quick select modifiers; large font which is too big but better than default, which is tiny on 23" 1080p screen but ok on 15" Retina). Spotlight fits this perfectly imo, but Lacona has much more functionality (though a side preview panel would be brilliant to show image previews/file metadata/path of files Lacona suggests, which I can't seem to do). 

Lists are much easier to scan in Spotlight as all icons and result text items line up, and they're grouped with section dividers. Showing all result icons on the right side would let items be scanned quickly, but I realise this isn't possible when you consider "and" with multiple actions/inputs.

Showing the top result faded inside the main text field, or just showing the icon of the top result on the right side of it, as Spotlight does, would save having to skim read the list. This would be especially useful when there is only 1 result returned by Lacona ("enable w" only shows enable wifi etc.).

Nitpicks..

A few more pixels of space to the right of token icons (e.g. between an app icon and the app's name) in search results would also improve spacing a bit (looks a bit cramped atm).

Perhaps match the system and use blue as the default search result highlight color, switch to gray if user has enabled graphite in system preferences.

Animating popovers for calculations/defining words when hovering over result items would smooth the jarring pop-in & match the rest of macOS.

**Comment by sebj** [comment]
Old & suggested designs (worth viewing at full size/100%).

Quick select modifier is hidden for now, could easily fit on right. Font size is 1 or 2px higher than Lacona's, probably too big but just messing around.

![Mockup](https://cloud.githubusercontent.com/assets/379991/20585223/677ac180-b1f1-11e6-83b8-e640914b1c7c.png)
